src/index.html

```diff
@@ -7,8 +7,69 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="nav__logo">
+        <img src="\images\logo.png" alt="logo">
+      </a>
+
+      <nav class="navbar">
```

Comment:
nav is a block, other elements inside are its children

```suggestion
      <nav class="nav">
```